framework-plugins/lisk-framework-faucet-plugin/src/plugin/faucet_plugin.ts

```diff
@@ -175,9 +193,9 @@ export class FaucetPlugin extends BasePlugin {
 
 		const config = {
 			applicationUrl: this._options.applicationUrl,
-			amount: (BigInt(this._options.amount) / BigInt(10 ** 8)).toString(),
+			amount: this._options.amount,
 			tokenPrefix: this._options.tokenPrefix,
-			captcha: this._options.captcha,
+			captcha: this._options.captchaSecret,
```

Comment:
this should be sitekey to send it to the UI. secret only should be used on the server side